docs/development/plugins-job-types.md

```diff
@@ -1,269 +1,1013 @@
+            Help: "Total number of started requests calling Job (may not be finished yet)"})
+)
+
+var (
+    metricRequestsDone = promauto.NewCounter(
+        prometheus.CounterOpts{
+            Name: "requests_done",
+            Help: "Total number of finished requests calling Job (processed and done)"})
+)
+
+var (
+    metricEndpointRequestsStarted = promauto.NewCounter(
+        prometheus.CounterOpts{
+            Name: "endpoint_requests_started",
+            Help: "Total number of started requests calling Job (may not be finished yet)"})
+)
+
+var (
+    metricRequestDuration = promauto.NewHistogram(
+        prometheus.HistogramOpts{
+            Name:    "request_duration",
+            Help:    "Duration of model call",
+            Buckets: []float64{0.001, 0.0025, 0.005, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5, 0.75, 1. 0, 2.5, 5.0, 7.5, 10.0, 25.0, 50.0, 75.0, 100.0, 250.0, 500.0, 750.0, 1000.0}}) //+Inf is implicit
+)
+
+var (
+    metricLastCallTimestamp = promauto.NewGauge(
+        prometheus.GaugeOpts{
+            Name: "last_call_timestamp",
+            Help: "Timestamp (in seconds) of the last request calling Job"})
+)
+
+func MetricHandler() gin.HandlerFunc {
+    handler := promhttp.Handler()
+
+    return func(context *gin.Context) {
+        handler.ServeHTTP(context.Writer, context.Request)
+    }
+}
+```
+</details>
+
+`go_wrapper/openapi.go` handles the swaggerui dashboard:
+<details>
+  <summary>File `go_wrapper/openapi.go`</summary>
+
+```go
+package main
+
+import (
+    "net/http"
+    "os"
+    "text/template"
+
+    "github.com/gin-gonic/gin"
+)
+
+func MountOpenApi(router *gin.Engine, baseUrl string) {
+    jobName := os.Getenv("JOB_NAME")
+    jobVersion := os.Getenv("JOB_VERSION")
+
+    router.Any(baseUrl+"/static/openapi.json", func(c *gin.Context) {
+        tmplt := template.New("openapi.json")
+        tmplt, _ = tmplt.ParseFiles("./swaggerui/openapi.json")
+
+        context := map[string]string{
+            "jobName":    jobName,
+            "jobVersion": jobVersion,
+        }
+        tmplt.Execute(c.Writer, context)
+    })
+    router.Any(baseUrl+"/static/:filename", func(c *gin.Context) {
+        filename := c.Param("filename")
+        http.ServeFile(c.Writer, c.Request, "./swaggerui/"+filename)
+    })
+
+    router.Any(baseUrl+"/", func(c *gin.Context) {
+        http.ServeFile(c.Writer, c.Request, "./swaggerui/index.html")
     })
 }
 ```
 </details>
 
-`rust_wrapper/Cargo.toml` is the file with package dependencies used by Rust:
+`go_wrapper/go.mod` and `go_wrapper/go.sum` are the files with package dependencies used by Go:
 <details>
-  <summary>File `rust_wrapper/Cargo.toml`</summary>
-
-```ini
-[package]
-name = "rust_wrapper"
-version = "0.1.0"
-edition = "2021"
-
-[dependencies]
-actix-web = "4.0.1"
-serde = { version = "1.0.137", features = ["derive"] }
-log = "0.4.17"
-env_logger = "0.9.0"
-serde_json = "1.0.81"
+  <summary>File `go_wrapper/go.mod`</summary>
```

Comment:
Maybe we could point to the link https://github.com/TheRacetrack/plugin-go-job-type/blob/master/golang-job-type/go_wrapper/go.sum instead of copying this huge file here, which makes it also harder to maintain. 

Same with the `metrics.go` and `openapi.go` files which are non-obligatory for making something a job type.
I'm afraid that too complex job type documentation might scare off the developers.